docroot/sites/default/settings.platformsh.php

```diff
@@ -185,3 +188,8 @@
   // Set the connector configuration to the appropriate value, as defined by the formatter above.
   $config['elasticsearch_connector.cluster.' . $es_cluster_name]['url'] = $platformsh->formattedCredentials($relationship_name, 'drupal-elastic');
 }
+
+// Environment indicator settings.
+$config['environment_indicator.indicator']['name'] = 'Live';
```

Comment:
Use `ucfirst($platformsh->branch);` instead of Live label, this will be more useful example